usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -2,64 +2,76 @@ DROP VIEW IF EXISTS award_delta_view;
-            CASE WHEN defc.group_name = ('covid_19') THEN faba.transaction_obligated_amount else 0 END AS transaction_obligated_amount,
-            faba.disaster_emergency_fund_code,
-            s.reporting_fiscal_year,
-            s.reporting_fiscal_period
-        FROM financial_accounts_by_awards faba
-        INNER JOIN eligible_submissions s ON s.submission_id = faba.submission_id
-        INNER JOIN disaster_emergency_fund_code defc on defc.code = faba.disaster_emergency_fund_code
-        WHERE faba.disaster_emergency_fund_code IS NOT NULL
-    ),
-    fy_final_outlay_balances AS (
-        -- Rule: If a balance is not zero at the end of the year, it must be reported in the
-        -- final period's submission (month or quarter), otherwise assume it to be zero
-        SELECT faba.award_id, sum(faba.gross_outlay_amount_by_award_cpe) AS prior_fys_outlay,
-            faba.disaster_emergency_fund_code
-        FROM eligible_file_c_records faba
-        GROUP BY
-            faba.award_id,
-            faba.reporting_fiscal_period,
-            faba.disaster_emergency_fund_code
-        HAVING faba.reporting_fiscal_period = 12
-        AND sum(faba.gross_outlay_amount_by_award_cpe) > 0
-    ),
-     current_fy_outlay_balance AS (
-        SELECT
-            faba.award_id,
-            faba.reporting_fiscal_year,
-            faba.reporting_fiscal_period,
-            faba.disaster_emergency_fund_code,
-            sum(faba.gross_outlay_amount_by_award_cpe) AS current_fy_outlay
-        FROM eligible_file_c_records faba
-        GROUP BY
-            faba.award_id,
-            faba.reporting_fiscal_year,
-            faba.reporting_fiscal_period,
-            faba.disaster_emergency_fund_code
-        HAVING concat(faba.reporting_fiscal_year::text, lpad(faba.reporting_fiscal_period::text, 2, '0')) IN
-        (SELECT max(fyp) FROM closed_periods) AND sum(faba.gross_outlay_amount_by_award_cpe) > 0)
+eligible_submissions as (
+	select s.reporting_fiscal_year, s.reporting_fiscal_period, s.submission_id
+    from submission_attributes s
+    inner join closed_periods c
+    	on c.submission_fiscal_year = s.reporting_fiscal_year
+    	and c.submission_fiscal_month = s.reporting_fiscal_period
+    	and c.is_quarter = s.quarter_format_flag
+),
+eligible_file_c_records as (
+    select
+        faba.disaster_emergency_fund_code,
+        faba.treasury_account_id,
+        faba.object_class_id,
+        faba.award_id,
+        faba.transaction_obligated_amount,
+        faba.gross_outlay_amount_by_award_cpe,
+        s.reporting_fiscal_year,
+        s.reporting_fiscal_period,
+        s.submission_id
+    from financial_accounts_by_awards faba
+    inner join submission_attributes s on faba.submission_id=s.submission_id
+    inner join disaster_emergency_fund_code defc
+    	on defc.group_name = 'covid_19'
+		and defc.code = faba.disaster_emergency_fund_code
```

Comment:
This and further replacements below may have swung _too_ far in the new direction. Since this will only push COVID DEFCs to the ES index. The mock for Adv. Search _does_ only show filtering on covids (LMNOP), so this may be okay for now. 

If you want to keep it open for all DEFCs, I think the way you had it doing a sum of value based on CASE and grouping by award+defc may be the way to go.